README.md

```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
 
 This repository is a monorepo managed by Lerna. This means that all components are centrally managed here, even though we publish them to NPM as separate packages.
 
+## Commits
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/) to automatically version the components and update their changelog's.
+Feel free to use a [tool of your choice](https://www.conventionalcommits.org/en/about/#tooling-for-conventional-commits) to generate these commits.
+
+Example for a commit message
+```
+feat(button): add new color blue
+
+Closes #ticket-1234
+```
+
+For more examples, especially with BREAKING CHANGES, have a look at the [Conventional Commits Examples](https://www.conventionalcommits.org/en/v1.0.0/#examples).
+
```

Comment:
I asked for a second thing: show the automatic Changelog entry resulting from a conventional commit. Seems you forgot this part, did you?

Reply:
I thought one of the main motivations for conventional commits in our case here is to automate changelog entries... so an example would be highly relevant.

Reply:
It depends on the library which we wanna use to create Changelogs. So for now, we only want to force the Commits, later the Changelogs. Means I cannot give you an Example of an Changelog entry.

Reply:
I see.